fplus-lib/src/core/mod.rs

```diff
@@ -1262,6 +1269,19 @@ impl LDNApplication {
             );
             let parsed_app_file = serde_json::to_string_pretty(&app_file)
                 .map_err(|e| LDNError::Load(format!("Failed to pare into string: {}", e)))?;
+
+            let issue_reporter_handle =
+                if let Some(issue_reporter_handle) = app_file.issue_reporter_handle {
+                    Some(issue_reporter_handle)
+                } else {
+                    get_issue_reporter_handle(&app_file.id).await.map_err(|e| {
```

Comment:
This should never be needed. If we have different values in app file and in the dedicated column, things are very broken already.